packages/modules/src/modules/banners/common/BannerWrapper.tsx

```diff
@@ -149,32 +149,26 @@ const withBannerData = (
         };
     };
 
-    const clickHandlerFor = (componentId: string) => {
+    const clickHandlerFor = (componentId: string, close: boolean) => {
```

Comment:
Hi Tom. If the current functionality is to not invoke onClose then maybe we should make false the default value for the close argument? `close = false`

Reply:
I think I'd prefer devs to have to think about whether or not a new cta should close the banner. The risk with default values for flags like this is it gets missed